### PR TITLE
Feat/inkwell phonemisation

### PR DESCRIPTION
**Pull request by JoelSaleem**
If you want to get this branch running locally, in the values.yaml file, change

`config.AIRFLOW__KUBERNETES__GIT_BRANCH: "main"` to `config.AIRFLOW__KUBERNETES__GIT_BRANCH: "feat/inkwell-phonemisation"`

and

`git.ref: main` to `git.ref: feat/inkwell-phonemisation`

### CODE REVIEW

File: tasks/check_16_bit.py [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+
+        if numbits == '16':  # OR IS IT AN INT?
+            pass
+
+
+if __name__ == "__main__":
+    if not should_skip_task('check_16_bit'):  # TODO: MAKE THIS A CONST?
+        run()
+    else:
+        print('skipping: check_16_bit')
```
Comment (JoelSaleem) [inline tasks/check_16_bit.py:35]: wip

File: tasks/check_16_bit.py [hunk 2]
```diff
@@ -0,0 +1,35 @@
+import subprocess
+from utils import (
+    build_client_s3,
+    get_filename_from_key,
+    should_skip_task
+)
+from os import listdir, chdir, makedirs
+
+
+def run():
```
[the remark this answers is not in the document]
Comment (udoniyor) [reply tasks/check_16_bit.py:10]: At the moment this function runs but don't error out if the wav is not 16bit, right? That is when downsampling comes in?

File: tasks/download_txts.py [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+
+def run():
+    client = build_client_s3()
+
+    txt_keys = []
+    with open('/usr/foo/data/txt_keys.txt') as f:
+        txt_keys = f.readlines()
+
+    stero_files = []
+    for key in txt_keys:
```
Comment (udoniyor) [inline tasks/download_txts.py:16]: At some point we should re evaluate the way the data is stored. Downloading these txts one by one is not efficient

File: tasks/get_inkwell_version.py [hunk 4]
```diff
@@ -0,0 +1,24 @@
+import subprocess
+from time import sleep
+from utils import (
+    should_skip_task
+)
+
+
+def run():
+    # TODO: CHANGE DOCKERFILE TO ONE LEVEL OF INKWELL
+    inkwell_version = subprocess.run(
```
Comment (udoniyor) [inline tasks/get_inkwell_version.py:10]: We could execute this while building the docker image, and store it as an Environment variable

File: tasks/phonemise_text.py [hunk 5]
```diff
@@ -0,0 +1,60 @@
+import subprocess
+import argparse
+from os import path, makedirs, chdir, listdir
+from time import sleep
+from shutil import copyfile
+
+
+def run(ignore_oov, inkwell_version, language):
+    chdir('/usr/inkwell')
+    print(
```
Comment (udoniyor) [inline tasks/phonemise_text.py:10]: Could switch to a logger, and use different levels i.e. debug for the command run, and info for what is happening

**Review by udoniyor — APPROVED** [review]

**Pull request merged**